dev/WindowsAppSDK.ApiScan.Cpp.props

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
could be out of the scope of this PR. we may hook this up in Directory.build.props so that it applies to all binaries. Then of course it cannot be cpp only.

Reply:
Hm, that's a good idea. I'm not as familiar with the C#/Managed settings though, so I'd prefer to leave that to a separate PR later on.